crates/telio-sockets/src/protector/apple.rs

```diff
@@ -466,6 +501,12 @@ pub fn setup_network_path_monitor() {
         };
 
         *INTERFACE_NAMES_IN_OS_PREFERENCE_ORDER.lock() = names.borrow().clone();
```

Comment:
Could we add a `telio_log_info!(...)` which would log what interfaces has been reported by the event and in what order. During in-branch testing, it would be nice to verify, that notifications are in-fact coming in a manner that we expect.